tests/Microsoft.Bot.Expressions.Tests/BadExpressionTests.cs

```diff
@@ -292,19 +291,25 @@ public class BadExpressionTests
             Test("foreach(items, item, item2, item3)"), // should have three parameters
             Test("foreach(items, add(1), item)"), // Second paramter of foreach is not an identifier
             Test("foreach(items, 1, item)"), // Second paramter error
+#if false
             Test("foreach(items, x, sum(x))"), // third paramter error
+#endif
             Test("select(hello, item, item)"), // first arg is not list
             Test("select(items, item)"), // should have three parameters
             Test("select(items, item, item2, item3)"), // should have three parameters
             Test("select(items, add(1), item)"), // Second paramter of foreach is not an identifier
             Test("select(items, 1, item)"), // Second paramter error
+#if false
             Test("select(items, x, sum(x))"), // third paramter error
+#endif
             Test("where(hello, item, item)"), // first arg is not list or structure
             Test("where(items, item)"), // should have three parameters
             Test("where(items, item, item2, item3)"), // should have three parameters
             Test("where(items, add(1), item)"), // Second paramter of where is not an identifier
             Test("where(items, 1, item)"), // Second paramter error
+#if false
             Test("where(items, x, sum(x))"), // third paramter error
+#endif
             Test("indicesAndValues(items, 1)"), // only one param
```

Comment:
Did you remove these errors?

Reply:
these were similar things, the functions no longer fail because we get back default(t) results. They used to throw exceptions..Thoughts?

---
In reply to: [367284959](https://github.com/microsoft/botbuilder-dotnet/pull/3253#discussion_r367284959) [](ancestors = 367284959)